app/models/spree/gateway/stripe_connect.rb

```diff
@@ -9,12 +9,6 @@ class StripeConnect < Gateway
 
       attr_accessible :preferred_enterprise_id
 
-      CARD_TYPE_MAPPING = {
```

Comment:
Isn't this needed anywhere for Stripe Connect? Are these changes to the old Stripe Connect code going to break it?

Reply:
no, when adapting the new method I found this is dead code in the existing method.
in the commit message for this deletion you can see the commit when the code became dead:
https://github.com/openfoodfoundation/openfoodnetwork/pull/4672/commits/785e89624cf02b084afd26cb3c9e116d4252664c